src/backend/services/synthesizer.py

```diff
@@ -0,0 +1,55 @@
+from google.cloud.texttospeech import (
+    AudioConfig,
+    AudioEncoding,
+    SynthesisInput,
+    TextToSpeechClient,
+    VoiceSelectionParams,
+)
+from googleapiclient.discovery import build
+
+from backend.config import Settings
+
+
+def synthesize(text: str) -> bytes:
+    """
+    Synthesizes speech from the input text.
+
+    Args:
+        text (str): The input text to be synthesized into speech.
+
+    Returns:
+        bytes: The audio content generated from the input text in MP3 format.
+    """
+    client = TextToSpeechClient(client_options={
+        "api_key": Settings().google_cloud.api_key
+    })
+
+    langauge = detect_language(text)
+
+    response = client.synthesize_speech(
+        input=SynthesisInput(text=text),
+        voice=VoiceSelectionParams(language_code=langauge),
+        audio_config=AudioConfig(audio_encoding=AudioEncoding.MP3)
+    )
+
+    return response.audio_content
+
+
+def detect_language(text: str) -> str:
+    """
+    Detect the language of the given text.
+
+    Args:
+        text (str): The text for which the language needs to be detected.
+
+    Returns:
+        str: The language code of the detected language (e.g., 'en', 'es').
+    """
+    if text.isascii():
+        return "en"
```

Comment:
I don't think this assumption can be made, for example the phrase: `J'aime manger des pommes` is ascii, but is french. Let's remove this if

Reply:
Yeah, that's a good point, thanks!